src/sha1.c

```diff
@@ -176,6 +176,32 @@ void Hash_Final() {
   }
 }
 
+WASM_EXPORT
+void Hash_Save() {
```

Comment:
With a `WASM_EXPORT uint8_t* Hash_GetState() { return context; }` function it would be more simple.